code/controllers/subsystem/garbage.dm

```diff
@@ -29,7 +29,7 @@ SUBSYSTEM_DEF(garbage)
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
 	init_order = INIT_ORDER_GARBAGE
 
-	var/list/collection_timeout = list(5 MINUTES, 10 SECONDS) // deciseconds to wait before moving something up in the queue to the next level
+	var/list/collection_timeout = list(GC_QDEL_IN_CUTOFF, 10 SECONDS) // deciseconds to wait before moving something up in the queue to the next level
```

Comment:
I feel like you should change the name to make more sense in this or maybe add a second define and have one use the other